python/docs/pyspark.sql.rst

```diff
@@ -7,6 +7,7 @@ Module Context
 .. automodule:: pyspark.sql
     :members:
     :undoc-members:
+    :exclude-members: builder
```

Comment:
This blocks the default dummy doc for `builder`

Reply:
I believe this is a bug in Sphinx. I think it should automatically detect the attribute docstring. I recently took a look before:

- `https://github.com/sphinx-doc/sphinx/pull/2299`
- `https://github.com/sphinx-doc/sphinx/issues/2298`

I haven't tested the fix by myself but I believe this will be fixed in Sphinx 1.6.6.